docs/getting-started/tutorial-visualizing.asciidoc

```diff
@@ -154,15 +157,17 @@ image::images/tutorial-visualize-map-2.png[]
 
 . Navigate the map by clicking and dragging.  Use the controls
 on the left to zoom the map and set filters.
-. *Save* this map with the name `Map Example`.
+. In the application toolbar, click *Save*.
+. Enter `Map Example` for the title.
+. Click *Save*.
 
 [float]
 [[tutorial-visualize-markdown]]
 === Markdown
 
 The final visualization is a Markdown widget that renders formatted text.
 
-. Create a *Markdown* visualization.
+. Return to the Visualization app and create a *Markdown* visualization.
```

Comment:
It might be better to change the order.  That is create the markdown before the map so the user doesn't have to return to the Visualization app.